app/services/core_data_connector/import/instances.rb

```diff
@@ -0,0 +1,135 @@
+module CoreDataConnector
+  module Import
+    class Instances < Base
+      def cleanup
+        super
+
+        execute <<-SQL.squish
+          UPDATE
+          core_data_connector_instances
```

Comment:
For these `UPDATE` statements, could we put the table names on the same line as `UPDATE`?